portfolio/src/main/webapp/index.html

```diff
@@ -119,6 +115,18 @@ <h2>Fun Facts</h2>
                 <li>Cutting down on my screentime</li>
                 <li>Going on hikes around town</li>
             </ul>
+
+            <form action="/data" method="POST">
+
+              <p>Have something to say? Let me know:</p>
+              <textarea name="text-input">Post your comment here!</textarea>
+              <br/><br/>
```

Comment:
Avoid br tags (especially several in a row).
Wrap items in divs or other elements if you need to create linebreaks, and use css if you need to position things with greater spacing

Reply:
Fixed!